model/src/w3wavemd.F90

```diff
@@ -697,24 +697,20 @@ SUBROUTINE W3WAVE ( IMOD, ODAT, TEND, STAMP, NO_OUT &
         FACX   =  1.
         END IF
 !
-#ifdef W3_SBS
-      NDSOFLG = 99
-#endif
-#ifdef W3_MPI
       SBSED = .FALSE.
-#endif
-#ifdef W3_SBS
-      SBSED = .TRUE.
-#endif
+      if (w3_sbs_flag) then
```

Comment:
Can this be kept as #ifdef W3_SBS and keep changing any #if-defs to a seperate PR?

Reply:
The issue here is that the sbs if-def is one of those "branching ifdefs" which are very difficult to code around w/o nesting additional ifdefs, which we wanted to avoid.

Reply:
It'd be great to back this out if possible to keep different development separate.

Reply:
No, it is not really possible to back out the change to this branching ifdef. I believe it would work to remove the wav_shr_flag file and put in a local definition of this flag.

Reply:
I think that sounds like a great path forward!

Reply:
It turns out I cannot remove wav_shr_flags completely since it is also used by the mesh cap code. I removed it from w3wavemd.F90 and added locally defined flags.

Reply:
Thanks @DeniseWorthen